src/engine/strat_engine/liminal/identify.rs

```diff
@@ -591,6 +591,7 @@ mod tests {
         )
         .unwrap();
 
+        #[allow(clippy::redundant_closure)]
```

Comment:
Plz remove any of the redundant closures that you can remove in this file.

Reply:
I just cherry-picked the commit from when I tried that the last time... and it worked; hooray.

Reply:
Great, keep going and remove all allows that you can remove, I _think_ it should be all.

Reply:
Ah yes, I have to remove the allow comments now.

Reply:
I think this is as far as I can go.  The only ones left are the dbus_api ones, that show "expected signature" errors if I try to remove the allow.